scripts/install_backend_python_libs.py

```diff
@@ -440,15 +450,6 @@ def verify_pip_is_installed():
                 'https://github.com/oppia/oppia/wiki/Installing-Oppia-%28'
                 'Windows%29')
         raise ImportError('Error importing pip: %s' % e)
-    else:
```

Comment:
Why is this deleted?

Reply:
This was used to downgrading pip (the comments are confusing), because the newest version of pip didn't support Python 2, this is not needed anymore.

Reply:
Wait -- from my reading of it, this would (and should) also trigger if pip was at a higher version than what we pinned it as in requirements.in, right? So wouldn't this still apply?

Reply:
Right, not in requirements.in though, in install_prerequisites.sh.

Reply:
Refactored.